src/Microsoft.ML.CodeGenerator/CodeGenerator/CSharp/TransformGenerators.cs

```diff
@@ -330,4 +330,26 @@ public override string GenerateTransformer()
         }
     }
 
+    internal class ImageLoading : TransformGeneratorBase
+    {
+        public ImageLoading(PipelineNode node) : base(node)
+        {
+        }
+
+        internal override string MethodName => "LoadRawImageBytes";
+
+        public override string GenerateTransformer()
+        {
+            string inputColumn = InputColumns.Count() == 1 ? InputColumns[0] : throw new Exception($"output columns for the suggested transform: {MethodName} are null");
+            string outputColumn = OutputColumns.Count() == 1 ? OutputColumns[0] : throw new Exception($"output columns for the suggested transform: {MethodName} are null");
+            string inputFolder = Properties["imageFolder"] as string;
```

Comment:
why dont we just use `pipelineNode.property`, the refactor in `TransformGeneratorBase` seems to be unnecessary